### ISSUE TITLE
Strange loss curves when training MB-MelGAN on custom multi-speaker dataset

### ISSUE THREAD

**Issue by samuel-lunii** [TensorSpeech/TensorFlowTTS#570]
Hi !
I am trying to learn models from scratch on my own multi-speaker dataset. 
Until now, Tacotron2 training seems to work fine, according to tensorboard data : 
![image](https://user-images.githubusercontent.com/40658675/118799653-42811a00-b89f-11eb-99c8-58d7d4d8574e.png)

However, I observed a strange behaviour for MB-MelGAN : 
![image](https://user-images.githubusercontent.com/40658675/118799839-7fe5a780-b89f-11eb-8b8f-b2618bceacca.png)
![image](https://user-images.githubusercontent.com/40658675/118799917-9855c200-b89f-11eb-91f5-a003165cffa6.png)

Note that training on LJSpeech dataset works fine. I did not change much to the preprocessing part (only `speaker_id.npy` files generation, not used for MB-MelGAN), and did not change anything to the `train_multiband_melgan.py` code.

Any clue about what's happening ? Am I doing something wrong with preprocessing ?

**Comment by dathudeptrai** [comment]
@samuel-lunii https://github.com/TensorSpeech/TensorFlowTTS/issues/247

**Comment by samuel-lunii** [comment]
Thank you @dathudeptrai 
Reducing the learning rate solved the problem :)
